test/models/ToolpadEditor.ts

```diff
@@ -113,7 +113,7 @@ export class ToolpadEditor {
   }
 
   waitForOverlay() {
-    const hudOverlay = this.appCanvas.getByTestId('hud-overlay');
+    const hudOverlay = this.appCanvas.getByTestId('page-overlay');
```

Comment:
in the ToolpadEditor model there is a `pageOverlay` property, maybe we can use it here?